src/components/AnchorForCommentsOnly/index.js

```diff
@@ -1,22 +1,20 @@
 import React from 'react';
+import {View} from 'react-native';
 import * as anchorForCommentsOnlyPropTypes from './anchorForCommentsOnlyPropTypes';
 import BaseAnchorForCommentsOnly from './BaseAnchorForCommentsOnly';
 import * as DeviceCapabilities from '../../libs/DeviceCapabilities';
 import ControlSelection from '../../libs/ControlSelection';
 import styles from '../../styles/styles';
 
 const AnchorForCommentsOnly = (props) => (
-    <BaseAnchorForCommentsOnly
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...props}
-        onPressIn={() => DeviceCapabilities.canUseTouchScreen() && ControlSelection.block()}
-        onPressOut={() => ControlSelection.unblock()}
-        // HTML renderer root view display is flex. Using flex will force all child elements
```

Comment:
Removed this comment because actually we can use the display inline, so I think the comment is not valid anymore.